starknet_py/proxy_check.py

```diff
@@ -4,6 +4,7 @@
 from starknet_py.net.models import parse_address
 
 
+# noinspection PyUnresolvedReferences
```

Comment:
Why is this needed here?

Reply:
Otherwise `"Client"` was showing as unresolved reference for some reason, and we cannot import it here due to circular reference

Reply:
I love circular references in python <3